admin/tests/test_integration.py

```diff
@@ -51,74 +51,6 @@
 v3_onion_services: true
 '''
 
-WHEN_BOTH_TRUE = '''app_hostname: app
```

Comment:
I'm not suggesting we should remove those integration tests altogether; I've just removed them for now to observe that all other tests still pass. If the overall approach looks good, I'll add integration tests specifically for the behavior in legacy mode, which should broadly resemble the old tests.